docs/pages/enterprise/workflow/ssh-approval-jira-cloud.mdx

```diff
@@ -136,6 +145,34 @@ By default, the Jira Teleport plugin will use a config in `/etc/teleport-jira.to
 </TabItem>
 </Tabs>
 
+<Details title="Running Teleport in Kubernetes?">
```

Comment:
Kubernetes users may not see this Details box before they create the config file mentioned above. I think we should make the Kubernetes option as visible as the option to run an executable on the host. 

I think we have two possibilities here:

1. Remove the "Self-Hosted" tab since, as far as I understand it, plugins can use the identity file to connect to the Auth Service (since they can extract the TLS credentials), but can't use the TLS credentials to connect to the Auth Service through the Proxy (because this requires SSH credentials). 

    After that, rename the "Cloud" tab to "Executable", then add a "Helm" tab. For this to make sense, we'd need to introduce the choice between deployment environments in a sentence above the `Tabs` component.
    
    We would also need to remove the "Self-Hosted" option in "Export the access-plugin certificate".

1. This hasn't been documented yet, but you can include a dropdown menu in a `Tabs` component so you can add multiple dimensions. There can be an "Executable" dropdown option with the current `TabItem`s and a `Helm` option with the `TabItem`s in the `Details` box. The current documentation is in [the comments for `Tabs`](https://github.com/gravitational/docs/blob/45d8ab1d9ecde58b0aa128e9d271281190e29878/components/Tabs/Tabs.tsx#L16-L58).

   In the second option, we would also need a sentence before the `Tabs` component introducing that there are two possible deployment environments and each has different config options for Cloud/Self-Hosted deployments.

Reply:
I've moved these things around to accomodate the Executable/Helm idea. What do you think? Is it easy to follow now?